src/DataAccess/DbalCommentRepository.php

```diff
@@ -33,9 +33,20 @@ function( Spenden $spenden ) {
 					->setAuthorName( $spenden->getName() )
 					->setCommentText( $spenden->getKommentar() )
 					->setDonationAmount( (float)$spenden->getBetrag() )
-					->setPostingTime( $spenden->getDtNew() );
+					->setPostingTime( $spenden->getDtNew() )
+					->setDonationId( $spenden->getId() )
+					->freeze()
+					->assertNoNullFields();
 			},
-			$this->entityRepository->findBy( [ 'isPublic' => true ], null, $limit )
+			$this->getSpenden( $limit )
+		);
+	}
+
+	private function getSpenden( int $limit ): array {
```

Comment:
I've stuck with Spenden for with the intention of this being changed when we switch to the renamed entities. For this method it really does not matter much to me, as it is private. Feel free to change it already if you prefer that.